app/javascript/src/require_communication_preference.js

```diff
@@ -33,7 +33,7 @@ $('document').ready(() => {
     displayPopUpIfPreferencesIsInvalid()
   })
 
-  smsToggle.addEventListener('change', () => {
+  smsToggle?.addEventListener('change', () => {
```

Comment:
This is unrelated to my current code here but was popping up an error in the console.